tournaments/index.js

```diff
@@ -960,6 +960,7 @@ class Tournament {
 					Economy.logTransaction(Chat.escapeHTML(winner) + ' has won ' + firstMoney + ' ' + (firstMoney === 1 ? global.currencyName : global.currencyPlural) + ' from a tournament.');
 				});
 			});
+			SG.addExp(wid, this.room, Math.round(tourSize));
```

Comment:
`wid` and `rid` are strings, don't you need to pass a user object to `SG.addExp`?